src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 16px;
+
+  transition-duration: $transition-duration;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    margin: 16px auto 0;
+  }
+
+  &__name {
+    font-weight: 500;
+    margin-top: 40px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover .card__name {
+    color: #34568b;
+  }
+
+  &__code {
+    font-size: $secondary-font-size;
+    line-height: $secondary-line-height;
+    color: $secondary;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__revievs {
+    font-size: $secondary-font-size;
+    line-height: $secondary-line-height;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price {
+    color: $secondary;
+  }
+
+  &__money {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__link {
+    box-sizing: border-box;
+    display: block;
+    width: 166px;
+    padding: 12px 70px;
+
+    transition-duration: $transition-duration;
+
+    color: $white;
+    text-align: center;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    margin-top: 16px;
+
+    border-radius: 5px;
+    background: $Blue-accent;
+    border: 1px none $Blue-accent;
+
+    &:hover {
+      background: none;
+      border-style: solid;
```

Comment:
```suggestion
    border: 1px solid $Blue-accent;

    &:hover {
```
Just make the border solid by default and do nothing on hover. In that case, it will not affect other elements
![hover](https://github.com/mate-academy/layout_catalog/assets/96409563/4e69806c-8670-48e6-8663-8c4cd43a8dda)